tests/end_to_end/test_account.py

```diff
@@ -26,6 +27,39 @@ async def erc20_token(deploy_solidity_contract, new_account):
     )
 
 
+@pytest.fixture(scope="function")
+async def cleanup(invoke, class_hashes):
+    yield
+    await invoke(
+        "kakarot",
+        "set_account_contract_class_hash",
+        class_hashes["account_contract"],
+    )
+    await invoke(
+        "kakarot", "set_cairo1_helpers_class_hash", class_hashes["Cairo1Helpers"]
+    )
+
+
+async def assert_transfer_success(erc20_token, new_account, other):
```

Comment:
maybe use the `Counter.inc()` instead as this is not really minimal to illustrate the fact that the original tx is well processed even though we actually upgrade the classe